pgtrigger/core.py

```diff
@@ -388,20 +388,20 @@ def __init__(
         if not self.name:
             raise ValueError('Trigger must have "name" attribute')
 
-        if len(self.name) > 53:
-            raise ValueError(f'Trigger name "{self.name}" > 53 characters. ')
+        if len(self.name) > 43:
+            raise ValueError(f'Trigger name "{self.name}" > 43 characters. ')
```

Comment:
Since a hash is appended to installed triggers, the trigger name given by the user must be shorter. PG only allows up to 63 characters in identifiers